<launch>

    <node pkg="alarm_device" type="new_alarm_device.py" name="new_alarm_device" respawn="true">
        <param name="Serial_Port" value="/dev/ttyUSB5 " type="string" />
    </node>

    <node pkg="alarm_device" type="alarm_control_node" name="alarm_control_node">
        <param name="gas_limit" type="int" value="10" />
        <param name="temp_limit" type="int" value="60" />
        <param name="alarm_volume" type="int" value="30" />
    </node>

</launch>
